desks-web-app/src/stores/booking-module.tsx

```diff
@@ -94,7 +94,7 @@ const getters: GetterTree<BookingsState, RootState> = {
                 return false
             }
             const key = `${roomId}-${moment(date).format('YYYY-MM-DD')}`
-            const weekKey = `${roomId}-${moment(date).startOf('week').format('YYYY-MM-DD')}`
+            const weekKey = `${roomId}-${moment(date).format('YYYY-MM-DD')}`
             return (state.loadingByRoomAndDate.has(key) && state.loadingByRoomAndDate.get(key))
               || (state.loadingByRoomAndWeek.has(weekKey) && state.loadingByRoomAndWeek.get(weekKey))
```

Comment:
loadingByRoomAndWeek that naming isn't really describing what you are using it for any more. You might need to adjust the data-structure that holds the loading state to better suit the need to representing the loading state for arbitrary time frames.